src/main/java/edu/SlowestKeySolution.java

```diff
@@ -1,9 +1,78 @@
 package edu;
 
+import java.util.Arrays;
+
 public class SlowestKeySolution {
     public static final char INVALID = '-';
 
+
+//    String keysPressed = "asdagrehhg";
+//    int[] releaseTimes = new int[] { 1, 3, 4, 5, 8, 9, 10, 13, 14, 17};
+//        SlowestKeySolution.solve(releaseTimes, keysPressed);
+
     public static char solve(int[] releaseTimes, String keysPressed) {
-        return INVALID;
-    }
-}
+        if (keysPressed == null || keysPressed.equals("")) {
+            return INVALID;
+        }  else if (releaseTimes == null) {
+            return INVALID;
+        } else if (!(releaseTimes.length == keysPressed.length())) {
+            return INVALID;
+        }
+        char[] ch = new char[keysPressed.length()];
+                for (int i = 0; i < keysPressed.length(); i++) {
+                    ch[i] = keysPressed.charAt(i);
+                }
+                System.out.println(ch);
+        int  b = (int) ch[0];
+        char tempChar2 = '0';
+        System.out.println(b);
```

Comment:
Task does not require to print out anything.

Reply:
Yes 💯. Still didn't finish.

Reply:
Cool :) If I understand correctly, in your situation PR was created too early :)

Reply:
![image](https://user-images.githubusercontent.com/40486932/99429575-dc780200-2910-11eb-9e34-5544770485d3.png)
You can create PR early too, just make sure it's marked as draft to indicate work in progress

Reply:
ok ^^